scripts/build_backend.py

```diff
@@ -64,7 +64,7 @@ def custom_build_cmake_clib(
 
     if iface in ["dpc", "spmd_dpc"]:
         if IS_WIN:
-            cxx = "icx"
+            cxx = "icx-cl"

```

Comment:
how come icx-cl? so far my builds with icx seem to work...

Reply:
No actual reason or difference, it was added during debugging. I'm going to remove it.